scripts/create-default-storage-class

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cat <<EOF | kubectl create -f -
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
Is there any particular reason why you use `v1beta1` and not `v1`? [Reference to the k8s wiki](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storageclasses)